<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="kor_input">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px dotted rgb(255, 0, 255);
background-color: rgb(170, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="eng_input">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>461</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px groove rgb(255, 0, 127);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="trans_btn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>130</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>번역실행</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>구글 한줄 번역기</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Condensed</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 12pt &quot;Bahnschrift Condensed&quot;;</string>
    </property>
    <property name="text">
     <string>한국어 입력</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>영어</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>430</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>중국어</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="jap_input">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>461</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px groove rgb(255, 0, 127);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>320</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>일본어</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="chn_input">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>460</y>
      <width>461</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px groove rgb(255, 0, 127);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="init_btn2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>전체초기화</string>
    </property>
   </widget>
   <widget class="QPushButton" name="init_btn1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>입력초기화</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>644</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
